models/labels/addresses/social/usage/lens/poster_frequencies/labels_lens_poster_frequencies.sql

```diff
@@ -1,65 +1,52 @@
-{{config(alias = alias('lens_poster_frequencies')
-
-    ,post_hook='{{ expose_spells(\'["polygon"]\',
+{{config(
+    alias = alias('lens_poster_frequencies')
+    , tags=['dunesql']
+    , post_hook='{{ expose_spells(\'["polygon"]\',
                                     "sector",
                                     "labels",
                                     \'["scoffie"]\') }}')
 }}
 
-WITH lens_addresses as 
-
-(SELECT to as address
-       ,handle as name
-       ,CAST (profileId AS string) AS profileId
-FROM {{ source('lens_polygon','LensHub_evt_ProfileCreated') }} pc 
-
-
-
+WITH lens_addresses as (
+    SELECT pc."to" as address
+       , handle as name
+       , CAST (profileId AS VARCHAR) AS profileId
+    FROM {{ source('lens_polygon','LensHub_evt_ProfileCreated') }} pc
 )
 
-,post_data as
- (
+,post_data as(
     SELECT
          output_0 as post_id
-         ,CAST(get_json_object(vars, '$.profileId') AS string) as profileId
- 
+         ,json_query(vars, 'lax $.profileId' omit quotes) as profileId
```

Comment:
i think there's a cleaner function for this, i've seen rob do often in nft spells
i believe it's this one [here](https://github.com/duneanalytics/spellbook/pull/3797/files#diff-aa56433c7f61d9d966f662da158eef661754370dd49c74e1e77057374187946eR13)
`json_extract_scalar`

Reply:
this applies to other spots in this model / PR too

Reply:
Fixed.